Formula/kubeless.rb

```diff
@@ -0,0 +1,32 @@
+class Kubeless < Formula
+  desc "Kubernetes Native Serverless Framework"
+  homepage "https://github.com/kubeless/kubeless"
+  url "https://github.com/kubeless/kubeless/archive/v0.2.3.tar.gz"
+  sha256 "e1b1c3b9498ae42b9f2ca7530506fcb37a136bdffb2af7957c3041a93502972a"
+  bottle :unneeded
+
+  depends_on "go" => :build
+
+  def install
+    commit = "bfefb40"
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/kubeless/kubeless").install Dir["*"]
+    cd "src/github.com/kubeless/kubeless" do
+      system "go", "build", "-o", bin/"kubeless", "-ldflags", "-w -X github.com/kubeless/kubeless/version.VERSION=v#{version} -X github.com/kubeless/kubeless/version.GITCOMMIT=#{commit}", "github.com/kubeless/kubeless/cmd/kubeless"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<-EOS.undent
+    To use kubeless you need a running Kubernetes cluster
```

Comment:
Kinda feel like this should just be using a `depends_on "kubernetes-cli" => :recommended` dep.

Reply:
True, the `kubernetes-cli` is recommended but what the caveat try to say is that a cluster should be ready and how to install the server side of Kubeless

Reply:
Generally Homebrew leaves basic usage instructions to the upstream READMEs to handle, not least because people barely read `caveats`. I'll let ILZ or another maintainer chime in on that though.

Reply:
The caveats should be removed.

Reply:
okay, I will remove them then